libvips/colour/profile_load.c

```diff
@@ -66,10 +66,31 @@ vips_profile_fallback_get( const char *name, size_t *length )
 
 	for( i = 0; (fallback = vips__profile_fallback_table[i]); i++ ) 
 		if( g_ascii_strcasecmp( fallback->name, name ) == 0 ) {
-			if( length )
+			void *data;
+			GConverter *converter;
+			GConverterResult res;
+			gsize bytes_read;
+			gsize bytes_written;
+			GError *error;
```

Comment:
You need to init this to NULL. Though I think I'd just pass NULL to g_converter_convert(), since you're not using the error message from GError.